src/module/system/text-editor.ts

```diff
@@ -370,7 +385,14 @@ class TextEditorPF2e extends TextEditor {
                           })
                           .join(" ");
                 html.innerHTML = inlineLabel ?? skillLabel;
-                html.setAttribute("data-pf2-check", params.type);
+                if (params.action) {
+                    const actionSlug = sluggify(params.action, { camel: "bactrian" });
+                    html.setAttribute("data-pf2-action", actionSlug);
```

Comment:
Pretty assigning `dataset` properties over setting attributes

Reply:
Yup, working on it, cleaning up the others in there while I'm at it.